pyveg/src/gee_interface.py

```diff
@@ -30,20 +30,23 @@
 LOGFILE = os.path.join(TMPDIR, "failed_downloads.log")
 
 
+
+
 # EXPERIMENTAL Cloud masking function.  To be applied to Images (not ImageCollections)
-def mask_cloud(image, input_coll):
+def apply_mask_cloud(image, input_coll):
     """
     Different input_collections need different steps to be taken to filter
     out cloud.
     """
-    if "LANDSAT" in input_coll:
-        mask_func = cloud_mask.landsat8ToaBQA()
-        return mask_func(image)
+    if input_coll=='LANDSAT/LC08/C01/T1_SR':
+        mask_func = cloud_mask.landsat8SRPixelQA()
+        image = image.map(mask_func)
```

Comment:
Ah, cool, so this "map" applies the function to all Images in the ImageCollection - nice! :)